### DISCUSSION TITLE
Offline libmesh build dies at netcdf

### DISCUSSION THREAD

**Discussion by mntbighker** [idaholab/moose#21304] [Q&A Getting Started] [answered]
make[2]: Leaving directory '/home/mwmoorcroft/moose/libmesh/build/contrib/gzstream'
Making all in netcdf/v4
make[2]: Entering directory '/home/mwmoorcroft/moose/libmesh/build/contrib/netcdf/v4'
make[2]: *** No rule to make target 'all'.  Stop.
make[2]: Leaving directory '/home/mwmoorcroft/moose/libmesh/build/contrib/netcdf/v4'
make[1]: *** [Makefile:1034: all-recursive] Error 1
make[1]: Leaving directory '/home/mwmoorcroft/moose/libmesh/build/contrib'
make: *** [Makefile:32937: all-recursive] Error 1

**Comment by mntbighker** [comment, marked as answer]
Oops, I didn't have the cmake or boost modules loaded. Could be the reason.

Nope, made sure cmake and boost were loaded, but it still dies at the same place.

**Reply by GiudGiud** [reply]
can you run the diagonistics script in moose/scripts and attach the log please

**Reply by mntbighker** [reply]
Once it finishes my latest compile attempt I'll do that. I tried loading the OpenHPC netcdf module.

**Reply by mntbighker** [reply]
[diag.txt](https://github.com/idaholab/moose/files/8904492/diag.txt)

**Reply by milljm** [reply]
is this related to this: https://github.com/idaholab/moose/discussions/21241#discussioncomment-2950845?

**Reply by mntbighker** [reply]
Yes, it seemed like I had moved past the original issues. This seems now like a new issue. I notice that netcdf has a lot of curl libs and IP addresses etc in the built config. But this being an offline build with no internet access I'm not sure if it should even be building netcdf? I'm not exactly sure why the build dies there.

**Reply by GiudGiud** [reply]
netcdf is useful to read files, especially hdf5 exodus files. 

Is this solved?
